src/Orckestra.Composer.CompositeC1/Providers/ScopeProvider.cs

```diff
@@ -11,7 +15,9 @@ public class ScopeProvider : IScopeProvider
         public IWebsiteContext WebsiteContext;
         public ISiteConfiguration SiteConfiguration;
 
-        public ScopeProvider(IWebsiteContext websiteContext, ISiteConfiguration siteConfiguration)
+        public ScopeProvider(IWebsiteContext websiteContext, 
+            ISiteConfiguration siteConfiguration
```

Comment:
If there are no changes in this file, it is better to rolback this modifications